### PR TITLE
Updating role lists

### PR DESCRIPTION
**Pull request by Danielx64**
Ok this is just the start as I am sure that I will find more roles, should I list every role from Regular to Hyper uber regular?

In relations to https://github.com/Microsoft-community/Community-website/issues/58

### CODE REVIEW

**Comment by Jasius** [comment]
> I list every role from Regular to Hyper uber regular?

Yeah as they have different perks assigned to them, I think they're actually documented somewhere however.

**Comment by Jasius** [comment]
They're listed as it is https://msft.chat/mod/docs/#roles @Danielx64

File: member/index.md [hunk 1]
```diff
@@ -16,14 +16,41 @@ We'd like you to bear in mind that none of our **moderators** work for nor are a
...
 | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | 🌟 Moderators         | Overlooking the community and its development; ensuring the safety of the members and the server itself.                                                                                                                                                                                                                                         |
+|   - Lead moderators                        | Be a very good moderator for a very long time and have Jaska love you |
```
Comment (Jasius) [inline member/index.md:24]: Member page shouldn't be updated, I'd prefer to have roles list short and concise here as it was, only mod page roles list should be comprehensive list.

Comment (Danielx64) [reply member/index.md:24]: That's fine, I'll rollback the page then

Comment (Danielx64) [reply member/index.md]: Ok, it rolled back to how it was,

**Pull request merged**